\documentclass[twoside,10pt,twocolumn]{article}


%%%%% THIS IS THE SECTION WHERE THE AUTHOR PUTS IN ALL OF THEIR TITLE AND AFFILIATION
%%%%% INFORMATION AND  A FEW OTHER THINGS
\newcommand{\myTitle}{Long-term genetic stability of Chinook salmon populations in the Trinity River, CA}

% redefine this to make author list. Note affiliation symbols are done manually.
% All caps tends to look better
\newcommand{\myAuthors}{ ANDREW P. KINZINGER$^{*,\S}$, {\sc ANTHONY J. CLEMENTO}$^{\dagger,\ddag}$, and J. CARLOS GARZA$^{\dagger,\ddag}$}

% redefine to make the affiliation list.  Note symbols are done manually
\newcommand{\myAffiliations}{$^*$Humboldt State University, Department of Fisheries Biology, 1 Harpst Street, Arcata, CA, 95521, $^\dagger$Fisheries Ecology Division, 
    Southwest Fisheries Science Center, National Marine Fisheries Service, NOAA,
    110 Shaffer Road, Santa Cruz, CA 95060, USA, $^\ddag$Institute of Marine Sciences, University of California, Santa Cruz, 110 Shaffer Road,Santa Cruz, CA 95060, USA}

% the email address for the corresponding author
\newcommand{\myEmailAddress}{Andrew.Kinziger@humboldt.edu }
\newcommand{\myEmailFootnote}{$^\S$}

% here you can put your very own copyright notice
\newcommand{\myCopyright}{\copyright US Federal Government work in the public domain in the USA}

% here you can put a running title (a short title that goes on the left of the 
% even pages)
\newcommand{\myRunningTitle}{Long-term genetic stability}

% and here you put the running author (short listing of authors for the
% upper right header on the odd pages)
\newcommand{\myRunningAuthor}{Kinzinger {\em et al.}}

%%%% DONE WITH AUTHOR/TITLE/ETC INFORMATION DEFINITION


% ------
% Fonts and typesetting settings
\usepackage[sc]{mathpazo}
\usepackage[T1]{fontenc}
\linespread{1.05} % Palatino needs more space between lines

\usepackage{microtype}
% ------
% Page layout
\usepackage[labelfont=bf,labelformat=simple,labelsep=quad]{caption}

\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{epstopdf}
\usepackage{amsfonts}
\usepackage{natbib}
\usepackage{subfigure}
\usepackage{pdfsync}
\usepackage{xspace}
\usepackage{mathrsfs}
\usepackage{fancyhdr}
\usepackage{cuted}
\usepackage{flushend}


%% some handy things for making bold math
\def\bm#1{\mathpalette\bmstyle{#1}}
\def\bmstyle#1#2{\mbox{\boldmath$#1#2$}}
\newcommand{\thh}{^\mathrm{th}}


%% Some pretty etc.'s, etc...
\newcommand{\cf}{{\em cf.}\xspace }
\newcommand{\eg}{{\em e.g.},\xspace }
\newcommand{\ie}{{\em i.e.},\xspace }
\newcommand{\etal}{{\em et al.}\ }
\newcommand{\etc}{{\em etc.}\@\xspace}



%% the page dimensions
\textwidth = 6.5 in
\textheight = 9 in
\oddsidemargin = -0.01 in
\evensidemargin = -0.01 in
\topmargin = -0.7 in
\headheight = 0.25 in
\headsep = 0.25 in
\parskip = 0.0in
\parindent = 0.25in

\setlength{\columnsep}{.4in}


%% change section heading styles
\makeatletter
\renewcommand\section{\@startsection {section}{1}{\z@}%
                                   {-3.5ex \@plus -1ex \@minus -.2ex}%
                                   {1.5ex \@plus 0ex}%
                                   {\normalfont\normalsize\bfseries}}
\renewcommand\subsection{\@startsection{subsection}{2}{\z@}%
                                     {-3.25ex\@plus -1ex \@minus -.2ex}%
                                     {1.5ex \@plus .2ex}%
                                     {\normalfont\normalsize\itshape}}
\makeatother

%% modify the abstract environment
\renewenvironment{abstract}
{\begin{quote}{\bf Abstract}\end{quote}\begin{quote}\bfseries\small}
{\end{quote}}


\fancypagestyle{firststyle}
{
   \fancyhf{}
   \chead[]{\vspace*{.1in}\includegraphics[width=\textwidth]{images/banner.pdf}}
%   \chead[]{\vspace*{.1in}}
   \lfoot[]{\footnotesize \myCopyright}
}

%% here is what I hope will be fancyplain
\fancyhead{} % clear all header fields
\fancyhead[LE]{{\bf \thepage}~~{\sl \myRunningTitle}}
\fancyfoot[RE,LO]{{\footnotesize \myCopyright}}
\renewcommand{\headrulewidth}{0pt}
\fancyhead[RO]{{\sl \myRunningAuthor}~~{\bf \thepage}}
\cfoot[]{}



%% Commands for some mathematical notation:
\newcommand{\bs}{\bm{s}}
\newcommand{\bbf}{\bm{f}}
\newcommand{\bx}{\bm{x}}
\newcommand{\bz}{\bm{z}}


\begin{document}



\pagestyle{fancyplain}
\thispagestyle{firststyle}



 \begin{strip}
   \mbox{}\\
   {\large\sc paper}
   \mbox{}\\
        {\LARGE\bf \myTitle \par}
    \mbox{}\\
    \uppercase{\myAuthors}\\ 
       \mbox{}\\
    {\em \myAffiliations}\\
    \mbox{}\\
    {\small \myEmailFootnote Correspondence: \myEmailAddress}
    
\begin{abstract}
We analyze historical samples collected over the last century to document the long-term genetic stability of Chinook salmon populations in the Trinity River, CA 
\end{abstract}
 \end{strip}

\section*{Introduction}
Historical spring and fall runs

Genetics and SNPs

Goals

\section*{Methods}
\subsection*{Study Site}
The Trinity River is found in northwestern California and is the largest tributary of the Klamath River system, which drains the Cascade/Klamath mountain range in southern Oregon.  Together, the two rivers constitute the second largest drainage in California after the Sacramento/San Joaquin system. The Klamath/Trinity historically maintained the third largest run of salmon on the West Coast, as well as steelhead, sturgeon and other important species(?). The river was free-flowing until the construction of Trinity Dam in 1960 and Lewiston Dam in 1963, which effectively eliminated anadromous access to over 100 miles of spawning and rearing habitat in the upper drainage while diverting millions of gallons of water into California's Central Valley for agriculture. Annual runs of salmon and steelhead were reported to have declined 90\% following the completion of the water project (Stene 1996). In response to declining salmon and steelhead runs, the Trinity River Hatchery was built to mitigate lost spawning habitat above the dam and to provide animals for harvest pursuant to Federally recognized tribal fishing rights. 
Chinook salmon in the Trinity River were proposed for listing under the Federal Endangered Species Act, but a NMFS status review in 1998 (Lindley?, Myers?)  determined that the listing was not warranted at that time.  However, a massive fish die-off in ???, and record low numbers since then, led to a subsequent petition to list in 2012, which was also denied (Federal Register 2012). The lower part of the river, below Lewiston Dam, was designated Wild and Scenic by the US Congress in 1981.

The Salmon River is another important tributary to the Klamath River, draining the high-elevation regions of the Trinity Alps and Marble Mountains. Unlike the Trinity River, the Salmon River has experienced comparatively fewer anthropogenic impacts. The watershed lies entirely in the Klamath National Forest and much of it is designated as a federal wilderness area. Most important to local fish populations however, is that the river is completely free-flowing, with no dams, large water projects or diversions. Similar to the lower Trinity River, the Salmon River was also included as part of the National Wild and Scenic Rivers system in 1981. Salmon populations that inhabit the river have been largely unaffected by the significant hatchery operations occurring elsewhere in the Klamath River system and throughout California and Oregon.

Spring and Fall runs?

\subsection*{Samples and Genotyping}
Historical and Salmon River samples were acquired from...
The samples included from 2011 were obtained by Hoopa Valley Tribal Fisheries at the Trinity River Hatchery as part of a broodstock collection effort.

DNA was extracted from scales and fin clips using silica-based Qiagen DNeasy Blood \& Tissue Kits in 96-well plates, according to the manufacturers recommended protocol.  For the 2011 samples, DNA was diluted 1:2 with distilled water, while for all other samples DNA was used at the extracted concentration (no dilution). 

Samples were genotyped with a panel of 96 single-nucleotide polymorphism markers (Clemento \etal 2014) using a Fluidigm EP1 system. The genotyping protocol for SNPtype$^\copyright$ chemistry recommends a multiplex preamplification step, particularly in situations where DNA concentration may be reduced or DNA quality may be poor.  Polymerase chain reaction (PCR) of the modern (2011) samples was run with the standard preamplification routine (15 thermal cycles of 95\ensuremath{^\circ}C for 15 seconds and 60\ensuremath{^\circ}C for 4 minutes), however we increased the number of cycles to 35 for the 1920s samples and to 25 for the remaining samples with the goal of increasing target copy number.  Preamplified DNA was then diluted 1:100 with distilled water and genotyped on 96.96 nanofluidic dynamic arrays. Genotypes were called using the Fluidigm SNP Genotyping Analysis software, version 3.1.3.

\subsection*{Analyses}
We genotyped 1488 individuals collected over the last century from the Trinity River. Data from the species-diagnostic locus (OkiOts\_120255-113) on the marker panel were excluded prior to analyses, and individuals with the coho allele were flagged for removal.  A total of 30 individuals were excluded due to missing data, defined here as successful genotypes at less than 30 markers.  In order to verify a Trinity River origin for each sample, all genotyped individuals were then assigned to their most likely population of origin using the software gsi\_sim (Anderson \etal 2008, Anderson 2010) and the baseline of Clemento \etal 2014.  This analysis confirmed the presence of two coho salmon in the dataset and identified a total of 14 out-of-basin fish (13 from the Rogue River in Oregon and one from the Smith River in northern California), all of which were removed prior to further analysis. It is important to note that in order to have our marker set perfectly overlap with this baseline, we removed an additional 4 loci (Ots\_Myc-366, Ots\_ALDBINT1-SNP1,  Ots\_112208-722, and Ots\_RAG3) from both datasets, leaving 91 SNP loci for all subsequent analyses.

Sample collections were analyzed for Hardy-Weinberg equilibriium (HWE) and linkage disequilibrium (LD) using Genepop on the Web, version 4.2 (Raymond and Rousset 1995, Rousset 2008). We employed the probability test for HWE, using the complete enumeration method (Louis and Dempster 1987) and default Markov chain parameters with 500 batches to reduce the standard error to acceptable levels (<0.02; Rousset 2008).  All pairs of loci were tested for LD using the log likelihood ratio statistic and default Markov chain parameters with 500 batches.  We estimated genetic differentiation (F$_{ST}$) between all pairs of populations (with $\theta$ of Weir and Cockerham 1984) using the software package GENETIX version 4.05 (Belkhir 1996-2004). Significance of F$_{ST}$ estimates was calculated with 1000 permutations of the dataset. 

We used multiple tools to assess the genetic relationships between the sample collections included in the analysis. Phylogeographic trees were constructed with Cavalli-Sforza and Edwards? (1967) chord distance (DCE) and the neighbor-joining algorithm in PHYLIP vers. 3.69 (Felsenstein 2005) and were visualized with DENDROSCOPE (Huson et al. 2007). Majority-rule consensus values were calculated from 10,000 bootstrap samples of the data using the PHYLIP component CONSENSE.  A subset of populations from the genetic baseline used for individual assignment was included in trees to provide a geographic context for the comparisons. 

Individual-based methods were also employed to examine the genetic relationships among our sample collections. We plotted the first four principal components in two dimensions using the {\em adegenet} package (Jombart and Ahmed 2011) in R version 3.1.0 (R Core Team 2014). Each component was plotted against each other component and the magnitude of the eigenvalues for each component calculated. We also used the software {\em structure} version 2.3.4 (Hubisz \etal 2009) to calculate the fractional ancestry of each individual from an assumed number of genetic clusters (K). Assumed number of clusters was examined over the range of 2-4, and each cluster was run five times with the results sorted and displayed using CLUMPP(Jakobsson and Rosenberg 2007) and {/em distruct} (Rosenberg 2004). For each {\em structure} run we used a burn-in of 50,000 repetitions and 150,000 data collection sweeps.




\section*{Results}
\subsection*{Population structure}

\subsection*{Bib}
%\bibliographystyle{men}
%{\footnotesize
%\bibliography{anderson_bibdesk}}

 \end{document}


